mino/minows/key/disk.go

```diff
@@ -0,0 +1,64 @@
+package key
+
+import (
+	"crypto/rand"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"go.dedis.ch/dela/core/store/kv"
+	"golang.org/x/xerrors"
+)
+
+// diskStorage provides persistent storage for private keys on disk.
+type diskStorage struct {
+	bucket []byte
+	db     kv.DB
+}
+
+func newDiskStore(db kv.DB) *diskStorage {
```

Comment:
Should be renamed to `newDiskStorage`, or with the other change `newDBStorage`.

Reply:
renamed to `NewStorage()`

Reply:
In fact it should be `KeyStorage`, because it only stores the key.